### PR TITLE
use Node 12.13.0 executor

### LINKED ISSUE #120: Update to work on Node 12
Since Cypress v3.5.0+ is using Node 12 and FF executor runs on Node12

### PR DESCRIPTION
**Pull request by bahmutov**
- installed server dependencies one by one, please use `npm ci` to install them locally
- closes #120

### CODE REVIEW

File: server/package.json [hunk 1]
```diff
@@ -23,8 +23,8 @@
     "boom": "7.x.x",
```
Comment (bahmutov) [inline server/package.json]: why isn't renovate pinning these?

File: renovate.json [hunk 2]
```diff
@@ -12,5 +12,11 @@
...
     "every weekend"
   ],
-  "masterIssue": true
+  "masterIssue": true,
+  "lockFileMaintenance": {
+    "enabled": true
+  },
+  "separatePatchReleases": true,
+  "separateMultipleMajor": true,
+  "rangeStrategy": "pin"
```
Comment (bahmutov) [inline renovate.json]: hope renovate starts pinning server package.json dependencies

**Comment by kevinold** [comment]
@bahmutov Ready for merge

**Comment by bahmutov** [comment]
done and done

**Pull request merged**